Mathlib/Algebra/Category/ModuleCat/Presheaf/Monoidal.lean

```diff
@@ -53,7 +53,8 @@ noncomputable def tensorObj : PresheafOfModules (R ⋙ forget₂ _ _) where
   map_id X := ModuleCat.MonoidalCategory.tensor_ext (by
     intro m₁ m₂
     dsimp [tensorObjMap]
-    simp)
+    simp
+    rfl) -- `ModuleCat.restrictScalarsId'App_inv_apply` doesn't get picked up due to type mismatch
```

Comment:
This file definitely gets worse off. Haven't figured out a way to make `simp` use `ModuleCat.restrictScalarsId'App_inv_apply`: replacing `rfl` with `erw [ModuleCat.restrictScalarsId'App_inv_apply, ModuleCat.restrictScalarsId'App_inv_apply]` also solves the goal.